back/src/Domain/Model/PostContent.php

```diff
@@ -1,21 +1,26 @@
 <?php
 
 namespace App\Domain\Model;
-
 use App\Domain\Exception\InvalidPostException;
 
 class PostContent
 {
     public function __construct(
         private readonly string $post,
```

Comment:
IMO your naming will lead to some complexity, you should avoid to use $post here since you already have an entity with this name